README.md

```diff
@@ -26,6 +26,11 @@ EMAIL_SMTP_HOST=<host of smtp server>
 EMAIL_SMTP_PORT=<port of smtp server>
 EMAIL_SERVER_USER=<username of email account>
 EMAIL_SERVER_PASS=<password of email account>
+EMAIL_ACCESS_TOKEN= <access token gor gmail api>
+EMAIL_REFRESH_TOKEN= <refresh token for gmail api>
+EMAIL_CLIENT= <client id>
+EMAIL_SECRET= <client secret>
+REDIRECT_URL = <redirect url>
```

Comment:
Is it required? If yes, can we hard code it?

Reply:
We can hard code client id , client secret , refresh token  but .. as this project is open source, one may use our client id and client secret for malpractices . I placed these in the .env file because of this reason only 😅 .

I can't remember why i put redirect url over there , needed it for something earlier but then later i forgot to remove it.

Reply:
Yes agreed. But I am just pointing at `REDIRECT_URL `

Reply:
that could be removed